workflows/test_validate.py

```diff
@@ -0,0 +1,130 @@
+"""
+Tests the functionality of validate.py with several different recipes which should raise different errors
+"""
+
+import pytest
+import mock
+import workflows
+from workflows.validate import validate_recipe, main
+import json
+import sys
+
+
+def test_no_file():
+    """Test that validate returns an error when it returns with no file"""
+    with pytest.raises(Exception):
```

Comment:
If you call the function without a parameter this should likely raise a `TypeError`, so check for the most specific exception, not `Exception`

And I suggest renaming the test
```def test_validate_returns_an_error_when_called_without_parameters():```
and dropping the comment.
Reasoning is that you never ever need to call the test, so the function name is only written once and therefore can be arbitrarily long. The function name is also what you get in the test failure overview, so might as well make it convey what the test is doing.